docs/src/pages/[platform]/connected-components/account-settings/change-password/react.mdx

```diff
@@ -1,3 +1,359 @@
+        ))}
+      </>
+    )
+  };
+}
+```
+
+    </ExampleCode>
+  </Example>
+
+  </ExpanderItem>
+  <ExpanderItem title="ErrorMessage" value="error-message">
+
+
+Props:
+
+    <ReactPropsTable props={ERROR_MESSAGE} />
+
+The following example overrides the error message component with [`Heading`](/components/heading) and [`Text`](/components/text) primitives.
+
+    <Example>
+      <ExampleCode>
+
+```jsx
+import { Heading, Text } from '@aws-amplify/ui-react';
+
+function App() {
+  const components = {
+    ErrorMessage: ({ children }) => (
+      <>
+        <Heading>Custom Error Message</Heading>
+        <Text variation="warning">{children}</Text>
+      </>
+    )
+  };
+  
+  return (
+    <AccountSettings.ChangePassword components={components} />
+  );
+}
+```
+
+      </ExampleCode>
+    </Example>
+  </ExpanderItem>
+  <ExpanderItem title="SubmitButton" value="submit-button">
+
+Props:
+
+    <ReactPropsTable props={SUBMIT_BUTTON} />
+
+You can reuse the `AccountSettings.ChangePassword.SubmitButton`:
+
+    <Example>
+      <ExampleCode>
+
+```jsx
+import { AccountSettings } from '@aws-amplify/ui-react';
+
+const components  = {
+  SubmitButton: (props) => (
+    <AccountSettings.ChangePassword.SubmitButton {...props}>
+      My Custom Button
+    </AccountSettings.ChangePassword.SubmitButton>
+  )
+}
+```
+
+      </ExampleCode>
+    </Example>
+
+or use [button primitive](/components/button) directly:
+
+    <Example>
+      <ExampleCode>
+
+```jsx
+import { AccountSettings, Button } from '@aws-amplify/ui-react';
+
+const components  = {
+  SubmitButton: (props) => <Button {...props}>My Custom Button</Button>
+}
+```
+
+      </ExampleCode>
+    </Example>
+
+The following example replaces `SubmitButton` with native html button:
+
+    <Example>
+      <ExampleCode>
+
+```jsx
+function App() {
+  const components = {
+    SubmitButton: ({ onClick, isDisabled }) => (
+      <button type="submit" onClick={onClick} disabled={isDisabled}>
+        Custom Button Text
+      </button>
+    )
+  };
+  
+  return (
+    <AccountSettings.ChangePassword components={components} />
+  );
+}
+```
+
+      </ExampleCode>
+    </Example>
+  </ExpanderItem>
+</Expander>
+
+### Theming
+
+`ChangePassword` component is built upon our robust [Amplify UI theming system](../../theming). To get started add a theme object and set the appropriate [design tokens](../../theming#design-tokens). You can then pass that `theme` to the `ThemeProvider` so the changes can take affect.
+
+<Example>
+  <ExampleCode>
+
+```jsx
+import { AccountSettings, Card, ThemeProvider } from '@aws-amplify/ui-react';
+
+const theme = {
+  name: 'custom-theme',
+  tokens: {
+    colors: {
+      border: {
+        primary: { value: '{colors.neutral.40}' },
+        secondary: { value: '{colors.neutral.20}' },
+        tertiary: { value: '{colors.neutral.10}' },
+      },
+    },
+    radii: {
+      small: { value: '2px' },
+      medium: { value: '2px' },
```

Comment:
Can `small` and `medium` be different sizes?

Reply:
I'll put medium to 3px!
```suggestion
      medium: { value: '3px' },
```